src/Symfony/Component/Yaml/Tests/ParserTest.php

```diff
@@ -443,7 +443,29 @@ public function testObjectSupportDisabledButNoExceptions()
      */
     public function testObjectsSupportDisabledWithExceptions()
     {
-        $this->parser->parse('foo: !!php/object:O:30:"Symfony\Tests\Component\Yaml\B":1:{s:1:"b";s:3:"foo";}', true, false);
+        $this->parser->parse('foo: !php/object:O:30:"Symfony\Tests\Component\Yaml\B":1:{s:1:"b";s:3:"foo";}', true, false);
+    }
+
+    /**
+     * @expectedException PHPUnit_Framework_Exception
+     * @expectedExceptionMessage !!php/object is deprecated use !php/object instead
```

Comment:
PHPUnit converts errors to exceptions, so this exception actually is thrown.
 Or isn't that what you mean?